hryhorieva/hw_5/src/main/java/dao/DocumentDao.java

```diff
@@ -0,0 +1,13 @@
+package dao;
+
+import entity.Document;
+import entity.User;
+
+import java.util.List;
+
+public interface DocumentDao {
+    List<Document> selectAll();
+    void insert(Document document, User user);
```

Comment:
It's strange, that to insert Document you need to add User here. The same for update